src/components/layout.js

```diff
@@ -9,11 +9,21 @@ import React from "react"
 import PropTypes from "prop-types"
 import { useStaticQuery, graphql } from "gatsby"
 import { GitHub, Twitter } from "react-feather"
+import "@deckdeckgo/highlight-code"
+import { defineCustomElements as deckDeckGoElement } from "@deckdeckgo/highlight-code/dist/loader"
 
 import Header from "./header"
 import "./layout.css"
 
+// Init Highlighting
+deckDeckGoElement()
+
 const Layout = ({ children }) => {
+  const foundedYear = "2020",
+    currentYear = new Date().getFullYear()
+  const year = `${
+    foundedYear == currentYear ? currentYear : foundedYear + "-" + currentYear
```

Comment:
Best practices ask us to use `===` instead of  `==`. Also on the else, we could use template literals, so instead of `foundedYear + "-" + currentYear` we could have `"${foundedYear }-${currentYear}"`